src/common/components/repositories-list/index.js

```diff
@@ -69,26 +70,28 @@ class RepositoriesList extends Component {
     }
   }
 
-  renderRow(snap) {
+  renderRow(snap, index) {
+    const { hasNamedSnaps, registerName, snapBuilds, authStore } = this.props;
     const { fullName } = parseGitHubRepoUrl(snap.git_repository_url);
 
     let latestBuild = null;
-    const snapBuilds = this.props.snapBuilds[fullName];
+    const snapBuild = snapBuilds[fullName];
```

Comment:
Name is confusing a bit, as this is a list of all builds for given snap.
But I get the confusion with `snapBuilds` from store (which is all builds for all snaps).

Maybe `currentSnapBuilds`?